overlord/ifacestate/ifacemgr_test.go

```diff
@@ -451,7 +451,7 @@ func (s *interfaceManagerSuite) TestDoSetupProfilesAddsImplicitSlots(c *C) {
 	// Ensure that we have slots on the OS snap.
 	repo := mgr.Repository()
 	slots := repo.Slots(snapInfo.Name())
-	c.Assert(slots, HasLen, 17)
+	c.Assert(slots, HasLen, 18)
```

Comment:
Can this be changed to say "more than 1" instead.

c.Assert(len(info.Slots) > 1, Equals, true)

Each new interface has to touch this